app/src/utils/concept-sort.js

```diff
@@ -0,0 +1,2 @@
+export const conceptScoreSort = (conceptArray) =>
```

Comment:
This file was deleted in favor of  conceptUtils.js
Check [this](https://github.com/aws-samples/medical-transcription-analysis/pull/58/files) out